yargs.js

```diff
@@ -661,6 +671,11 @@ function Yargs (processArgs, cwd, parentRequire) {
       if (~handlerKeys.indexOf(cmd) && cmd !== completionCommand) {
         setPlaceholderKeys(argv)
         return command.runCommand(cmd, self, parsed)
+      } else {
+        if (recommendCommands) {
+          const similiarCommand = didYouMean(cmd, handlerKeys)
+          console.log(__('Did you mean %s?', similiarCommand))
```

Comment:
We should probably move this out of the `argv._` loop to avoid outputting multiple "did you mean"s. I would think the first encountered non-null recommendation would take precedence.